src/locale/lang/fr.js

```diff
@@ -0,0 +1,85 @@
+export default {
+  el: {
+    datepicker: {
+      now: 'Maintenant',
+      today: "Aujourd'hui",
```

Comment:
First, thank you for contributing! Here's a thing: some of the texts here are nested in a limited space. Take "Today" for example:
![image](https://cloud.githubusercontent.com/assets/10095631/20352594/c17ffb22-ac52-11e6-933e-549f49905534.png)
I don't think there is enough space for "Aujourd'hui".
Does the French language have any abbreviation or shorthand for "Aujourd'hui" and other long texts you provided here?